src/opera/pge/dswx_s1/dswx_s1_pge.py

```diff
@@ -736,7 +780,7 @@ class DSWxS1Executor(DSWxS1PreProcessorMixin, DSWxS1PostProcessorMixin, PgeExecu
     LEVEL = "L3"
     """Processing Level for DSWx-S1 Products"""
 
-    SAS_VERSION = "0.2.1"  # Beta release https://github.com/opera-adt/DSWX-SAR/releases/tag/v0.2.1
+    SAS_VERSION = "0.3"  # Gamma release https://github.com/opera-adt/DSWX-SAR/releases/tag/v0.3
```

Comment:
Just wondering if the comment might be better placed in the docstring on the next line.